front/tests/pages/op-timetable-page-model.ts

```diff
@@ -0,0 +1,217 @@
+import { type Locator, type Page, expect } from '@playwright/test';
+
+import enTranslations from '../../public/locales/en/operationalStudies/scenario.json';
+import frTranslations from '../../public/locales/fr/operationalStudies/scenario.json';
+
+class OperationalStudiesTimetablePage {
+  readonly page: Page;
+
+  readonly trainCountText: Locator;
+
+  readonly invalidTrainsMessage: Locator;
+
+  readonly scenarioTimetableTrains: Locator;
+
+  readonly selectedScenarioTimetableTrain: Locator;
```

Comment:
nit: Maybe you can remove the Scenario in the name ?
```suggestion
  readonly timetableTrains: Locator;

  readonly selectedTimetableTrain: Locator;
```